.github/workflows/publication.yml

```diff
@@ -53,6 +53,27 @@ jobs:
       - name: Build with Maven
         run: ${{ env.BUILD_COMMANDS }}
 
+      - name: Setting up variables
+        run: |
+          echo "MAVEN_GROUP_ID=$(mvn org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.groupId -q -DforceStdout)" >> $GITHUB_ENV
+          echo "MAVEN_ARTIFACT_ID=$(mvn org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.artifactId -q -DforceStdout)" >> $GITHUB_ENV
+          echo "MAVEN_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.version -q -DforceStdout)" >> $GITHUB_ENV
+          REF_VERSION=${GITHUB_REF_NAME:1}
+          REF_VERSION=${REF_VERSION/+*}
+          echo "REF_VERSION=$REF_VERSION" >> $GITHUB_ENV
+
+      - name: Publish to Nexus
+        uses: sonatype-nexus-community/nexus-repo-github-action@1.0.0
+        with:
+          serverUrl: https://nexus-jx.apps.serv.run/
```

Comment:
This static value should be managed with a Variable: You can use NEXUS_INSTANCE_URI wich is a global organizazione variable.